drivers/infiniband/hw/ntrdma/ntrdma_eth.c

```diff
@@ -590,10 +590,9 @@ static void ntrdma_eth_rx_fill(struct ntrdma_eth *eth)
 			goto dma_err;
 		}
 
-		rc = ntc_req_signal(dev->ntc, eth->dma_chan, NULL, NULL,
-				NTB_DEFAULT_VEC(dev->ntc));
+		rc = ntc_signal(dev->ntc);
```

Comment:
@leonidravich comment is valid here too, if you request imm32 above, you need to request signal here